spec/requests/automate_workspaces_spec.rb

```diff
@@ -66,5 +99,35 @@
 
       expect(response).to have_http_status(:bad_request)
     end
+
+    it 'decrypt a password' do
+      api_basic_authorize action_identifier(:automate_workspaces, :decrypt)
+
+      post(api_automate_workspace_url(nil, aw.guid), :params => decrypt_params)
+
+      expect(response).to have_http_status(:ok)
+      expect(JSON.parse(response.body)['value']).to eq(password)
+    end
+
+    it 'decrypt collection of passwords' do
+      api_basic_authorize action_identifier(:automate_workspaces, :decrypt)
+
+      post(api_automate_workspace_url(nil, aw.guid), :params => decrypt_params_a)
+
+      expect(response).to have_http_status(:ok)
+      expect(JSON.parse(response.body)['results'][0]['value']).to eq(password)
```

Comment:
Can the order of these be guaranteed?

Also, prefer one expectation where possible:

```ruby
expected = {
  "results" => a_collection_containing_exactly( # assuming order can't be guaranteed
    a_hash_including("value" => password"),
    a_hash_including("value" => p45)
    # etc...
  )
}
expect(response.parsed_body).to include(expected)